packages/algolia/src/client.ts

```diff
@@ -10,6 +15,11 @@ export type EntityResponses = {
   [USER_ENTITY_TYPE]: UserResponse;
 };
 
+export type BatchRequest = {
+  action: BatchActionType;
```

Comment:
im not sure we want to expose Algolia's implementation like that here? After all the purpose of this client is to abstract away. Perhaps we want this to become a `saveMany` method or something like that instead?

Reply:
There is such feature in Algolia - like save many, (as far as I remember).

Reply:
i dont think you understand what i mean - at the moment what we want is an ability to save multiple objects yeah?

We built an abstraction to hide some of the implementation details to make the interface more dev friendly for us. So we shouldn't have to expose an `action` for a `batch` request and instead have a method that enables us to save multiple objects. Underneath it should call the batch method, imho